### DISCUSSION TITLE
Map data is not yet available

### DISCUSSION THREAD

**Discussion by dchristopherfennell** [pelicanmapping/osgearth#2286] [Support Q&A] [answered]
Hello,

We are using osgEarth 3.0 since we have to support CentOS 7. We build osgEarth to use the OpenGL core profile. When using ArcGIS, certain areas get the image "**Map data is not yet available**". However, if I run osgEarth 2.10, not built with the core profile, I get data at the same area. You have to be at ground level to see this and it does not happen everywhere.

I can get this when running tests/arcgisonline.earth with osgEarth 3.0 / osg365 both built with the OpenGL core profile. Go to Kodiak Island, Alaska and at ground level, you'll see "Map data is not yet available".

Any clues to why this is happening?

**Comment by gwaldron** [comment, marked as answer]
Well, I think the code that processes the "no-data image" got blown away in some refactoring or another. 
I will convert this to a defect and we'll look into it. Thanks.
